dist/Dockerfile.build/Dockerfile

```diff
@@ -32,3 +32,5 @@ RUN \
   mkdir -p $HOME/.cargo/git/ && \
   find $HOME/. -type d -exec chmod 777 {} \; && \
   find $HOME/. -type f -exec chmod ugo+rw {} \;
+
+WORKDIR /go/src/github.com/openshift/cincinnati
```

Comment:
Is this GOPATH structure as the CI is built mainly for Go packages, and it clones the source accordingly?

Reply:
Yes, that's the path CI would put source at. I don't think its required though - removed